docs/en/setup/backend/backend-storage.md

```diff
@@ -159,11 +159,18 @@ Please download MySQL driver by yourself. Copy the connection driver jar to `oap
 ```yaml
 storage:
   mysql:
+    properties:
+      jdbcUrl: ${SW_JDBC_URL:"jdbc:mysql://localhost:3306/swtest"}
+      dataSource.user: ${SW_DATA_SOURCE_USER:root}
+      dataSource.password: ${SW_DATA_SOURCE_PASSWORD:root@1234}
+      dataSource.cachePrepStmts: ${SW_DATA_SOURCE_CACHE_PREP_STMTS:true}
+      dataSource.prepStmtCacheSize: ${SW_DATA_SOURCE_PREP_STMT_CACHE_SQL_SIZE:250}
+      dataSource.prepStmtCacheSqlLimit: ${SW_DATA_SOURCE_PREP_STMT_CACHE_SQL_LIMIT:2048}
+      dataSource.useServerPrepStmts: ${SW_DATA_SOURCE_USE_SERVER_PREP_STMTS:true}
+    metadataQueryMaxSize: ${SW_STORAGE_MYSQL_QUERY_MAX_SIZE:5000}
 ```
-
-All connection related settings including link url, username and password
-are in `datasource-settings.properties`. 
-This setting file follow [HikariCP](https://github.com/brettwooldridge/HikariCP) connection pool document.
+All connection related settings including link url, username and password are in `application.yml`. 
+Here are some of the settings, please follow [HikariCP](https://github.com/brettwooldridge/HikariCP) connection pool document for all the settings.
```

Comment:
@tristaZero You should be familiar with HikariCP config and JDBC settings, right? More professional than me. Please put all the necessary and recommended setting items here. The user will change it later.

Reply:
So...the setting for dataSource..is not enough, or appropriate? Those settings are originally from `datasource-settings.properties`. From my side, those settings are enough for dataSource, if user need more setting items, he can refer to HikariCP document.

Reply:
If you think they are enough in most cases, I am good about this too. If anyone needs more, they can add by themselves, or send a pull request to us.